include/zrepl_prot.h

```diff
@@ -151,6 +153,7 @@ typedef struct mgmt_ack mgmt_ack_t;
 enum zvol_rebuild_status {
 	ZVOL_REBUILDING_INIT,		/* rebuilding can be initiated */
 	ZVOL_REBUILDING_SNAP,		/* zvol is rebuilding snapshots */
+	ZVOL_REBUILDING_AFS_TRANS,	/* rebuilding in transition to AFS */
```

Comment:
we have to add this to ZVOL_IS_REBUILDING macro also

Reply:
done.